### DISCUSSION TITLE
Unable to setup https (Docker) | results in "mixed content error"

### DISCUSSION THREAD

**Discussion by shakeabi** [firefly-iii/firefly-iii#6567] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I have a setup a firefly instance using docker and tried enabling Https for it. I've followed [this](https://docs.firefly-iii.org/firefly-iii/installation/docker/) for setup and it results in some content not being loaded. For ex: charts, bills to pay, per day etc.,
![image](https://user-images.githubusercontent.com/36559835/197799930-5b27b511-dbeb-4ad7-afd6-1ddf82859cd4.png)
The console error is 
```
Refused to connect to 'http://<domain>/chart/account/frontpage' because it violates the following Content Security Policy directive: "connect-src 'self'".
```

Note that only some requests use http and others load properly rightly by querying https.

### Expected behaviour

All content should be loaded through https.

### Debug information

Debug information generated at 2022-10-25 19:51:00 Asia/Calcutta for Firefly III version **5.7.13**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.13 |
| Firefly III API | 1.5.6 |
| Build | 284, 16-10-2022 16:14:10 CEST |
| DB version | 18 (exp. 18) |
| PHP | `8.1.11` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Asia/Calcutta |
| Browser TZ | Asia/Calcutta |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | debug, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2022-10-01 00:00:00 |
| Session end | 2022-10-31 23:59:59 |
| Session first | 2022-10-17 23:35:21 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36 |

**Comment by JC5** [comment]
Checkout these instructions, they may also work for you.

https://docs.firefly-iii.org/data-importer/install/self_hosted/#tls

**Reply by figadore** [reply]
where are these instructions these days? I'm using HAProxy and getting "Blocked: mixed-content" errors on the importer, and it is not possible to submit the login form on the main app.
> login:30 Mixed Content: The page at 'https://firefly-iii.example.com/login' was loaded over a secure connection, but contains a form that targets an insecure endpoint 'http://firefly-iii.example.com/login'. This endpoint should be made available over a secure connection.

I already have `TRUSTED_PROXIES=**` in both .env files
